robust-websocket.js

```diff
@@ -75,6 +77,12 @@
     }
 
     self.send = function() {
+      if (realWs.readyState !== self.OPEN) {
+        throw Object.assign(
+          new Error('An attempt was made to use a WebSocket that is not usable'),
+          { name: 'InvalidStateError' }
+        )
+      }
```

Comment:
Why would we try to re-implement the WebSocket standard? calling `realWs.send.apply(...)` should just throw this error.

Reply:
As previously stated - `realWs.send` is not always present

Reply:
But that's not what this code is checking. If you want to check for th3 existence of send and throw an Error, that is acceptible.